### PR TITLE
Succesfully run integration test by getting tv show object list from …

### PR DESCRIPTION
**Pull request by aaryan-jain**
…our API endpoint and added new fields in the tv show object.

### CODE REVIEW

File: src/main/java/org/incubyte/actors/PeopleService.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import java.util.Optional;
 @Singleton
 public class PeopleService {
-    private TmbdClient tmbdClient;
+    private final TmbdClient tmbdClient;
```
Comment (khitab-incubyte) [inline src/main/java/org/incubyte/actors/PeopleService.java:14]: Why is this added?

Comment (aaryan-jain) [reply src/main/java/org/incubyte/actors/PeopleService.java:14]: Hello, this is added because the value of tmdbClient need not be changed anywhere except  the constructor
